osutil/user.go

```diff
@@ -60,30 +60,35 @@ func AddExtraSudoUser(name string, sshKeys []string, gecos string) error {
 		return fmt.Errorf("adduser failed with %s: %s", err, output)
 	}
 
-	sudoersFile := filepath.Join(sudoersDotD, "create-user-"+name)
+	sudoersFile := filepath.Join(sudoersDotD, "create-user-"+strings.Replace(name, ".", ",", -1))
```

Comment:
s/2e/2E/ too.. both should work, but the uppercasing creates a nice visual distinction since we're only allowing lowercase letters in the username.